### PR TITLE
LPS-181449 | Expose the actions available in individual entities schemas

### PR DESCRIPTION
**Pull request by Hijky**
Background:
Add a testcase to expose the actions available in individual entities schemas

Test Plan
CanGetMethodAndHrefForKeywordActions
Given in Headless Admin Taxonomy of API ExplorerWhen I navigate to the Keyword SchemaThen I can see method and href values for all available to an individual Keyword element actions

CanGetMethodAndHrefForTaxonomyCategoryActions
Given in Headless Admin Taxonomy of API ExplorerWhen I navigate to the TaxonomyCategory SchemaThen I can see method and href values for all available to an individual TaxonomyCategory element actions

CanGetMethodAndHrefForTaxonomyVocabularyActions
Given in Headless Admin Taxonomy of API ExplorerWhen I navigate to the TaxonomyVocabulary SchemaThen I can see method and href values for all available to an individual TaxonomyVocabulary element actions

Jira ticket:
https://liferay.atlassian.net/browse/LPS-186139

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

**Comment by magjed4289** [comment]
ci:test:headless-functional:nocompile

**Comment by Hijky** [comment]
The failures are not related to my PR.

File: modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-test/src/testFunctional/tests/ExposeActionsForAvailableIndividualEntitiesSchemasOfHeadlessAdminTaxonomy.testcase [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCPNoSelenium();
+		}
+	}
+
+	@priority = 4
+	test CanGetMethodAndHrefForKeywordActions {
+		task ("When I navigate to the Keyword Schema") {
+			Click(
```
Comment (yanan) [inline modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-test/src/testFunctional/tests/ExposeActionsForAvailableIndividualEntitiesSchemasOfHeadlessAdminTaxonomy.testcase:32]: Just wondering if we need to create a related method in macro to expand all actions in Keyword schema, I mean something like this:
```
task ("When I navigate to the Keyword Schema") {
  APIExplorer.expandAllActionsInSchema(schema="Keyword", actionsList="");
}
```
 Let's see what @magjed4289 suggest us to do :smile:

File: modules/apps/headless/headless-discovery/headless-discovery-test/src/testFunctional/macros/APIExplorer.macro [hunk 2]
```diff
@@ -30,7 +30,7 @@ definition {
 			locator1 = "OpenAPI#SCHEMA_PROPERTY",
 			property = ${property},
 			schema = ${schema},
-			value1 = "${property} { method string default: ${method} href string default: ${portalURL}/o/c/${objectHref} }");
+			value1 = "${property} { method string default: ${method} href string default: ${portalURL}/o/${objectHref} }");
```
Comment (yanan) [inline modules/apps/headless/headless-discovery/headless-discovery-test/src/testFunctional/macros/APIExplorer.macro:33]: :+1:

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-functional:nocompile

File: modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-test/src/testFunctional/tests/ExposeActionsForAvailableIndividualEntitiesSchemasOfHeadlessAdminTaxonomy.testcase (hunk 1 above)
Comment (Hijky) [reply modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-test/src/testFunctional/tests/ExposeActionsForAvailableIndividualEntitiesSchemasOfHeadlessAdminTaxonomy.testcase]: I think it's a better way to use a macro to replace and I have modified the PR

**Review by magjed4289 — APPROVED** [review]

**Comment by magjed4289** [comment]
ci:forward

**Pull request closed without merging**